engine/schema/src/main/resources/META-INF/db/schema-41910to41920.sql

```diff
@@ -22,5 +22,20 @@
 -- Add last_id to the volumes table
 CALL `cloud`.`IDEMPOTENT_ADD_COLUMN`('cloud.volumes', 'last_id', 'bigint(20) unsigned DEFAULT NULL');
 
-ALTER TABLE `cloud`.`counter` DROP KEY `uc_counter__provider__source__value`;
-CALL `cloud`.`IDEMPOTENT_ADD_UNIQUE_KEY`('cloud.counter', 'uc_counter__provider__source__value_removed', '(provider, source, value, removed)');
+SELECT
```

Comment:
@Pearl1594 
maybe we can add a procedure similar as

https://github.com/apache/cloudstack/blob/main/engine/schema/src/main/resources/META-INF/db/procedures/cloud.idempotent_drop_foreign_key.sql

Reply:
good call @weizhouapache that will help with a lot of these parallel upgrade issues.